app/views/shelters/_form.html.erb

```diff
@@ -41,7 +41,7 @@
         <%= f.label :shelter %>
         <%= f.text_field :shelter, :placeholder => 'Name of Shelter (like Manatee Bay Middle School)', :required => true, :id => 'geocode-name' %>
         <%= f.label :phone %>
-        <%= f.text_field :phone, :placeholder => '(123) 123-4567', :required => true, :id => 'geocode-phone' %>
```

Comment:
I'm not sure we want the phone not to be required for shelters, as calling them is a major effort of the response.

Reply:
There are plenty of examples in the shelters data where the phone number was unknown at time of data entry.  We could include shelters with a blank phone in the "needs update" page to indicate a phone number needs to be located.

Reply:
OK, we can leave it as is.